### DISCUSSION TITLE
Netbox database error preventing it from starting.

### DISCUSSION THREAD

**Discussion by cryansharp** [netbox-community/netbox#11327] [Q&A] [answered]
We added a new device status to /opt/netbox/netbox/dcim/choices.py a few weeks ago and I then attempted to upgrade Netbox yesterday.  However, it appeared to have errored during the upgrade.  I tried to back out the changes and perform the upgrade again, but we're getting the following error when Netbox tries to load in a browser.  

**<class 'django.db.utils.ProgrammingError'>

column extras_customlink.content_type_id does not exist
LINE 1: ...k"."created", "extras_customlink"."last_updated", "extras_cu...
                                                             ^
Python version: 3.9.13
NetBox version: 3.3.9**

Is there a way to fix this error?  

This is set up as an AWS EC2 instance with a RDS instance for the database.  I have also done a restore of the RDS instance in the event that we need to restore the database, but don't have instructions for restoring the database in Netbox.  Thanks. 

![Screen Shot 2022-12-28 at 8 10 30 AM](https://user-images.githubusercontent.com/26879222/209817322-657333d8-ae85-45bb-9f60-95371e67dd90.png)

**Comment by markkuleinio** [comment, marked as answer]
Here is the link to documentation that says how you can define new status values: https://docs.netbox.dev/en/stable/configuration/data-validation/#field_choices

I don't know what you did and how the database was corrupted because of that. If you can find out how to revert the database changes, you can restore the database, revert any changes (in the database and in the code) and then continue using the database in NetBox.

**Reply by cryansharp** [reply]
Thank you for the documentation on how to define new status values.  I'll take a look to see what I did wrong on that part.  

I was able to fix my database problem.  I did a restore of my RDS instance and pointed /opt/netbox/netbox/netbox/configuration.py to the new DB name.  After a reboot, it's working as expected again.  

So, this should now be resolved.  I'll walk through the upgrade process again to see how it got corrupted in the first place.
